### PR TITLE
Adding emr (1) resource to v1beta1 version

### LINKED ISSUE #346: Moving emr (1) resource to v1beta1 version

### PR DESCRIPTION
**Pull request by mykolalosev**
### Description of your changes
Add 1 resources in the emr group:

- [x] [aws_emr_security_configuration](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/emr_security_configuration)

Fixes #346 

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested
`aws_emr_security_configuration` manually and uptest - https://github.com/upbound/provider-aws/actions/runs/3751651651

### CODE REVIEW

**Comment by mykolalosev** [comment]
/test-examples="examples/emr/securityconfiguration.yaml"

File: config/externalname.go [hunk 1]
```diff
@@ -1724,6 +1724,11 @@ var ExternalNameConfigs = map[string]config.ExternalName{
 	//
 	// Media Convert Queue can be imported via the queue name
 	"aws_media_convert_queue": config.NameAsIdentifier,
+
+	// emr
+	//
+	// EMR Security Configurations can be imported using the name
+	"aws_emr_security_configuration": config.NameAsIdentifier,
```
Comment (sergenyalcin) [inline config/externalname.go:1731]: Could you please remove this resource from the externalnamenottested.go file?

Comment (mykolalosev) [reply config/externalname.go:1731]: Done, thank you!

**Review by sergenyalcin — APPROVED** [review]
Thanks @mykolalosev LGTM!

**Pull request merged**